specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-01-01/domainservices.json

```diff
@@ -395,6 +395,11 @@
           "type": "string",
           "description": "The name of the Azure domain that the user would like to deploy Domain Services to."
         },
+        "deploymentId": {
```

Comment:
You shouldn't be making any changes to stable api-versions.  If you want to request an exception you must follow the breaking change policy outlined in the PR description

Reply:
@pilor  - We have a few properties that were missing in our swagger initially. We recently got S360 violations for the missing properties. If we leave the swagger as is, then our S360 will stay red indefinitely (or can we get some exception for this) and if we fix the swagger its a breaking change it seems. What is the suggested path forward?

Reply:
If your API has always been returning these properties and they were only missed in your swagger specs then there is no API change to review here.   Please work with the SDK reviewer @xccc-msft to determine if this will result in an SDK breaking change.

Reply:
@xccc-msft - Could you take a look at this? I am adding new properties that were missing before, adding default error responses, and changing a incorrect type for a field from datetime to string.

Reply:
As it's not released on maven, not a breaking change for Java SDK. I'm not sure about the other SDKs.

Reply:
@xccc-msft - could you either tell me what is blocking from your side or approve from your side?

Reply:
I'm not from ARM while we have to wait for ARM signoff. Thanks.